src/partials/bloblang-playground.hbs

```diff
@@ -311,7 +311,7 @@ function execute() {
       const parsedResult = JSON.parse(result);
 
       // Separate message and metadata
-      const message = parsedResult.msg || parsedResult;
+      const message = parsedResult.msg;
```

Comment:
We should get a `msg` field in here if `isValidJSON(result) == true`, but maybe there are some edge cases I'm not thinking of right now... Not sure.

The `parsedResult.msg || parsedResult` approach causes trouble if the mapping is just `root = "foobar"` for example.

Reply:
In this case, I'm seeing: `Error: failed to parse mapping: expected query, got: 'foob`

Reply:
Do we want to tackle this issue in this PR or should be open a separate Jira?

Reply:
https://deploy-preview-289--docs-ui.netlify.app/playground

Reply:
`root = “foobar”` works 🙂

looks like I was using the wrong quotes